test/common/suite.ini

```diff
@@ -0,0 +1,5 @@
+[default]
+core = app
+description = memcached common tests with #! using TAP
+script = common.lua
```

Comment:
There is no such Lua file and it is not used for an app server by test-run AFAIU.

Reply:
Right, removed.

Reply:
```diff
--- a/test/common/suite.ini
+++ b/test/common/suite.ini
@@ -1,5 +1,4 @@
 [default]
 core = app
 description = memcached common tests with #! using TAP
-script = common.lua
 is_parallel = False
```